todo_app/data/trello_items.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import os
+
+TRELLO_BASE = "https://api.trello.com/1"
+BOARD_ID = "63dd25f4eaaf087231fa1c44"
```

Comment:
It looks to me like this is never used, can we get rid of it in favour of the run-time loaded environment variables?